README.md

```diff
@@ -278,6 +279,22 @@ You have two Ceph authentication options available (http://kubernetes.io/v1.0/ex
    ```
    You will have to pass the Secret name (using `--ceph_secret` flag) when running the benchmakrs. In this case it should be: `--ceph_secret=my-ceph-secret`.
 
+### Mesos configuration
+Mesos provider communicates with Marathon framework in order to manage Docker instances. Thus it is required to setup Marathon framework along with the Mesos cluster. In order to connect to Mesos you need to provide IP address and port to Marathon framework using `--marathon_address` flag.
+
+Currently only local disks are supported so you need to execute Perfkit with `--scratch_disk_type=LOCAL` flag.
```

Comment:
You can make the default disk type local (in default_config_constants.yaml) so that you don't always have to specify this flag